calculator/calculator/calculatorButton/ButtonList1to+.swift

```diff
@@ -0,0 +1,237 @@
+    var body: some View {
+        HStack{
+            Button(action:{
+                
+                if CalculatorDataSheet.isPlus {
+                    CalculatorDataSheet.isActCalculation = "plus"
+                }
+                else if CalculatorDataSheet.isDivide {
+                    CalculatorDataSheet.isActCalculation = "divide"
+                }
+                else if CalculatorDataSheet.isMinus {
+                    CalculatorDataSheet.isActCalculation = "minus"
+                }
+                else if CalculatorDataSheet.isMultiply {
+                    CalculatorDataSheet.isActCalculation = "multiply"
+                }
+
+                if !CalculatorDataSheet.isCalculation{  // 연산 중이 아닐 때
+                    CalculatorDataSheet.isDivide = false
+                    CalculatorDataSheet.isAC = true
+                    if CalculatorDataSheet.viewNumS == "0"{
+                        CalculatorDataSheet.isDivide = false
+                        CalculatorDataSheet.viewNumS = "1"
+                    }
+                    else{
+                        let numberFormatter = NumberFormatter()
+                            numberFormatter.numberStyle = .decimal
+                            numberFormatter.maximumFractionDigits = 9
+                        CalculatorDataSheet.viewNumS = (numberFormatter.string(for: Double(CalculatorDataSheet.viewNumS.components(separatedBy: [","]).joined() + "1"))!)
+                    }
+                }
+                else {  //연산 중 일 때
+              
+                    if !CalculatorDataSheet.isFirstCalculation {  // 첫 연산 일 때
+                        CalculatorDataSheet.isFirstCalculation = true // 다음은 첫 연산이 아니라고 명시
+                        CalculatorDataSheet.viewNumS2 = "1"
+                        CalculatorDataSheet.isDivide = false
+                        CalculatorDataSheet.isPlus = false
+                        CalculatorDataSheet.isMinus = false
+                        CalculatorDataSheet.isMultiply = false
+                        
+                    }
+                    else {   //첫 연산이 아닐 때
+                        if CalculatorDataSheet.viewNumS2 == "0"{
+                            CalculatorDataSheet.viewNumS2 = "1"
+                        }
+                        else{
+                            let numberFormatter = NumberFormatter()
+                                numberFormatter.numberStyle = .decimal
+                                numberFormatter.maximumFractionDigits = 9
+                            CalculatorDataSheet.viewNumS2 = (numberFormatter.string(for: Double(CalculatorDataSheet.viewNumS2.components(separatedBy: [","]).joined() + "1"))!)
+                        }
+                    }
+                }
+            }
+            ){
+                Circle().frame( width: UIScreen.main.bounds.size.width /  5, height : UIScreen.main.bounds.size.height / 10).foregroundColor(.gray).overlay(Text("1").foregroundColor(.white))
```

Comment:
```suggestion
                Circle()
                .frame( width: UIScreen.main.bounds.size.width / 5, height : UIScreen.main.bounds.size.height / 10)
                .foregroundColor(.gray)
                .overlay(Text("1")
                .foregroundColor(.white))
```

적당한 줄바꿈이 있다면 가로스크롤 하지 않더라도 한눈에 코드를 파악하기 쉬워요 !